### PR TITLE
Added thank you style

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?
This is the thank you page for the bundles. The implementation was done using formstack. Unfortunately, it is really hard to style Forsmstack and I had to use many `!important` statements.

## Changes
* Create a CSS for the thank you page and apply the necessary styles.

### Desired

thankyou01

### Final Result 

thanyouuuuu

### CODE REVIEW

File: frontend/app/views/bundle/thankYou.scala.html [hunk 1]
```diff
@@ -8,20 +8,19 @@
...
     bundleVariant: BundleVariant,
-    selectedOption: String)(implicit token: play.filters.csrf.CSRF.Token)
+    selectedOption: String,
+    heroImage: model.OrientatedImages)(implicit token: play.filters.csrf.CSRF.Token)
 
 @main("Supporters", pageInfo=pageInfo) {
+    @* ===== First part ===== *@
+    @fragments.bundle.bundleHeader(
+        heroImage,
+        Html("Thank you for<br/>your interest"),
```
Comment (SiAdcock) [inline frontend/app/views/bundle/thankYou.scala.html]: `br` tags don't need to be self-closing

File: frontend/assets/stylesheets/components/_bundle-thank-you.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    .fsForm {
+        margin-top: 0 !important;
+        padding-top: 0 !important;
+    }
+
+    label {
+        font-size: 18px !important;
+        margin-left: 60px !important;
```
Comment (SiAdcock) [inline frontend/assets/stylesheets/components/_bundle-thank-you.scss]: `$gs-gutter * 3`

File: frontend/assets/stylesheets/components/_bundle-thank-you.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        width: 460px !important;
+        height: 36px;
+        margin-left: 60px;
+        border-radius: 18px !important;
+    }
+
+    input[type=submit] {
+        width: 460px !important;
+        height: 36px;
+        margin-left: 60px;
```
Comment (SiAdcock) [inline frontend/assets/stylesheets/components/_bundle-thank-you.scss]: `$gs-gutter * 3`

File: frontend/assets/stylesheets/components/_bundle-thank-you.scss [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        border-radius: 18px !important;
+    }
+
+    textarea {
+        border-radius: 18px !important;
+        width: 460px !important;
+        margin-left: 60px;
```
Comment (SiAdcock) [inline frontend/assets/stylesheets/components/_bundle-thank-you.scss]: `$gs-gutter * 3`

**Comment by svillafe** [comment]
It is working on **PROD**

**Pull request merged**